src/Credentials.js

```diff
@@ -229,8 +237,16 @@ class Credentials {
    * @param    {Object[]}    [opts.vc]           An array of JWTs about the requester, signed by 3rd parties
    * @returns  {Promise<Object, Error>}          A promise which resolves with a signed JSON Web Token or rejects with an error
    */
-  createVerificationSignatureRequest(unsignedClaim, {aud, sub, riss, callbackUrl, vc} = {}) {
-    return this.signJWT({unsignedClaim, sub, riss, aud, vc, callback: callbackUrl, type: Types.VERIFICATION_SIGNATURE_REQUEST})
+  createVerificationSignatureRequest(unsignedClaim, { aud, sub, riss, callbackUrl, vc } = {}) {
+    return this.signJWT({
+      unsignedClaim,
```

Comment:
Yes, done by prettier. I increased the preferred line width from 80 to 120 to get rid of other similar line splits. This one would be 129 wide on a single line and having 7 args in a row is a bit hard to visually parse at a glance. It kind of defeats the purpose of running something like prettier if we manually override individual lines, so maybe it would be most helpful to reaching a standard style by discussing this in terms of preferred config options?

Reply:
Yeah definitely agree, 120 line max seems reasonable to me!

And yeah I think automating the decisions is good, just was curious generally about if prettier makes all of those formatting changes itself